lib/jobs/dell-wsman-discovery.js

```diff
@@ -8,7 +8,7 @@ var di = require('di');
 module.exports = DellWsmanDiscoveryJobFactory;
```

Comment:
I think it's better to put wsman-discovery Refine in another PR(Including the unit test). Although this bios reset PR depends on wsman discover function to find a valid node. But from unit test perspective, there is no tight coupling.